### PR TITLE
Translation update 2023 02 15

### LINKED ISSUE #1190: Update translation to Norwegian (Engine)
The Norwegian translation (nb.po) needs update. Please see [Instructions for translators] and contact us if there are any issues.

Create pull request against `release-branch-v2022.2.2` branch, not `develop` branch, for this issue.

[Instructions for translators]: https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/Instructions-for-translators.md

### PR DESCRIPTION
**Pull request by repperille**
## Purpose

This PR updates and adds translations to Norwegian

## Context

fixes #1190 

## Changes

...

## How to test this PR

...

### CODE REVIEW

**Comment by matsduf** [comment]
@repperille, I am sorry for not being fully clear. Can you base your branch on the `release-branch-v2022.2.2` branch and then add your commit on that? As it is now, you have based it on develop branch (and that is what the instructions says) which means that other commits are included. Those are meant for the next major release.

File: share/nb.po [hunk 1]
```diff
@@ -2,15 +2,66 @@ msgid ""
...
+#, perl-brace-format
+msgid ""
+"Domain name has an ASCII label (\"{label}\") with a character not permitted."
+msgstr ""
+"Domenenavnet har et ledd (\"{label}\") i ASCII som innholder et ugyldig tegn."
+
+#, perl-brace-format
+msgid ""
+"Domain name has a non-ASCII label (\"{label}\") which is not a valid U-label."
+msgstr "Domenenavnet har et ledd (\"{label}\") som ikke er gyldig Unicode."
```
Comment (matsduf) [inline share/nb.po:42]: A label can be a valid Unicode string without being valid U-label. U-label is more restricted than Unicode string. There is, as I see, no god alternative to using the term U-label. 

Maybe:
* "Domenenavnet har et ledd" --> "Domenenavnet har et ikke-ASCII ledd"
* "ikke er gyldig Unicode" --> "ikke er en gyldig U-label"

File: share/nb.po [hunk 2]
```diff
@@ -517,6 +702,79 @@ msgstr ""
...
+msgid "Check for too short or too long RRSIG lifetimes"
+msgstr "Sjekk etter for korte eller for lange RRSIG-levetider"
+
+#. DNSSEC:DNSSEC05
+msgid "Check for invalid DNSKEY algorithms"
+msgstr "Sjekk etter ugyldige DNSKEY-algoritmer"
+
+#. DNSSEC:DNSSEC06
+msgid "Verify DNSSEC additional processing"
+msgstr "Verifier håndtering av RRSIGposter i \"Additional Section\""
```
Comment (matsduf) [inline share/nb.po:727]: "RRSIGposter" --> "RRSIG-poster"?

**Review by matsduf — CHANGES_REQUESTED** [review]

File: share/nb.po (hunk 1 above)
Comment (repperille) [reply share/nb.po:42]: Yes I agree that the
"ikke er gyldig Unicode" --> "ikke er en gyldig U-label"
is better

**Comment by repperille** [comment]
About the release-branch-v2022.2.2.
I have created another PR #1199 based on the release-branch-v2022.2.2 but I can not see that there are any differences from development??

**Comment by matsduf** [comment]
> About the release-branch-v2022.2.2. I have created another PR #1199 based on the release-branch-v2022.2.2 (...)

Thanks. You could have created a new branch with the same name and force-pushed.

> but I can not see that there are any differences from development??

The `release-branch-v2022.2.2` branch has the same content as the `master` branch. The `develop` branch has additional changes on top.

**Comment by matsduf** [comment]
Replaced by #1199

**Pull request closed without merging**